Bugfixes

cs-rohit-khune opened the pull request:
Mantis #0796534
- This global variable is automatically get created by the MITRE connector during connector configuration and used for data ingestion under "MITRE Framework" SP
- Hence removed the global variable 'mitre_version' from this SP as there is no use for it

Mantis #0781959
Updated info.json, made the following changes
- Updated connector entries as per the latest format(removed connector version and updated connectors\data.json)
- Updated recordset entry to the latest format(added module name and apiName)
- removed discarded keys such as 'exported_from' and 'exported_by'

cs-rohit-khune committed:
Bugfixes

Mantis #0796534
- This global variable is automatically get created by the MITRE connector during connector configuration and used for data ingestion under "MITRE Framework" SP
- Hence removed the global variable 'mitre_version' from this SP as there is no use for it

Mantis #0781959
Updated info.json, made the following changes
- Updated connector entries as per the latest format(removed connector version and updated connectors\data.json)
- Updated recordset entry to the latest format(added module name and apiName)
- removed discarded keys such as 'exported_from' and 'exported_by'

cs-suyog-jinturkar commented on info.json:
```diff
@@ -28,17 +28,15 @@
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-mitre-attack-threat-hunting/blob/develop/README.md",
+    "category": [],
     "supportInfo": "support@fortinet.com",
-    "help": "https:\/\/github.com\/fortinet-fortisoar\/solution-pack-mitre-attack-threat-hunting\/blob\/develop\/README.md",
```
Remove support@fortinrt.com

cs-rohit-khune replied:
replaced `support@fortinet.com` with `Fortinet Customer Support`

cs-suyog-jinturkar commented on info.json:
```diff
@@ -28,17 +28,15 @@
     "certified": "true",
     "publisher": "Fortinet",
     "description": "The solution pack demonstrates users a variety of scenarios and use cases around threat hunting based on using the information provided by the MITRE ATT&CK Framework.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-mitre-attack-threat-hunting/blob/develop/README.md",
```
Description should contain connector name as suggested by Amit

cs-rohit-khune replied:
Added connector name in SP description

cs-rohit-khune committed:
Updated info.json

- Added integration name in SP description
- Updated support info to "Fortinet Customer Support"
